corehq/apps/reports/standard/deployments.py

```diff
@@ -93,7 +93,8 @@ def _columns(self):
                              alt_prop_name='reporting_metadata.last_submission_for_user.commcare_version',
                              sql_col='last_form_app_commcare_version'),
             DataTablesColumn(_("Number of unsent forms in user's phone"),
-                             help_text=_("""The number of unsent forms corresponding to app in users' phones"""),
+                             help_text=_("""The number of unsent forms in users' phones for selected app version
```

Comment:
do we also select a specific app version or just app?

Reply:
@orangejenny any suggestions on text here?

Reply:
Is it possible to get the actual filter inputs so the text can say either "app version X" or "last built app"?

Reply:
umm, you mean change the text based on when user selects an app dynamically in the form and in case of pre-populated values?

Reply:
Yes, can you use the value of `self.selected_app_id` to decide what help text to show?

Reply:
I am fine with keeping that simple as well
`The number of unsent forms in users' phones for selected application or the recent application installed`

we don't have option to select version, right? So that should not say "app version". And I don't think the user would know what "built" means, so "installed" might be better, or "installed/updated".